FS22_HudColor/translations/l10n_fr.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" standalone="no" ?>
+<l10n>
+  <translationContributors>Rudolfet</translationContributors>
+  <elements>
+    <e k="mod_title" v="HudColor"/>
+    <e k="settings_hud_color_label" v="Couleur HUD"/>
+    <e k="settings_hud_color_label_needs_restart" v="(Il est nécessaire de redémarrer la sauvegarde !!)"/>
+    <e k="settings_hud_color_description" v="Changer la couleur du HUD pour un autre préréglage !!! Après le changement, vous devez redémarrer la sauvegarde !!!"/>

```

Comment:
Please add `.` after `préréglage`

Reply:
the exclamation mark is correct?

Reply:
yes, for now. I'll change it later. Thanks